cookbooks/mediawiki/templates/default/mw-ext-osmcalwikiwidget.inc.php.erb

```diff
@@ -0,0 +1,3 @@
+<?php
+# DO NOT EDIT - This file is being maintained by Chef
+wfLoadExtension( 'OSMCALWikiWidget' );
```

Comment:
There's no need for this if it's just going to call `wfLoadExtension` as we do that by default if a `mediawiki_extension` resource doesn't specify a configuration template.

Reply:
Thanks! I've overlooked that behaviour. Fixed.